<!DOCTYPE html>
<html>
<head>
  <title>Brownell Lab Meeting</title>
  <meta charset="utf-8">
  <meta name="description" content="Brownell Lab Meeting">
  <meta name="author" content="Kenny Daily">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
    <link rel="stylesheet" href = "assets/css/ribbons.css">

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
    <!-- END LOGO SLIDE -->
    

    <!-- TITLE SLIDE -->
    <!-- Should I move this to a Local Layout File? -->
    <slide class="title-slide segue nobackground">
      <hgroup class="auto-fadein">
        <h1>Brownell Lab Meeting</h1>
        <h2>Merkel Cell Polyomavirus Detection</h2>
        <p>Kenny Daily<br/></p>
      </hgroup>
          </slide>

    <!-- SLIDES -->
      <slide class="" id="slide-1" style="background:;">
  <hgroup>
    <h2>MCV genome</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell%20Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/MCV350_layout.png" alt="MCV350" title="MCV350"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Slide 1</h2>
  </hgroup>
  <article>
    <ol>
<li>MCC cells do not typically harbor episomal MCV viral DNA</li>
<li>MCV is clonally integrated in the tumor chromosomal DNA</li>
<li>Mutations resulting in truncation of large T antigen (LT)</li>
<li>Expression of full length LT promotes viral replication</li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Slide 2</h2>
  </hgroup>
  <article>
    <ul>
<li>Several studies show ~80% of MCCs have virus present</li>
<li>Mostly by PCR on FFPE tissues</li>
<li>CN variation</li>
<li>IHC to sT and LT reported as well (again, ~80% positive for LT)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Analysis of MCV Load</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Title.png" width="1000" alt="Rodig Title"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Hypothesis</h2>
  </hgroup>
  <article>
    <ol>
<li><p>It is widely thought that MCV is a causative oncogenic mechanism, but 20% of MCC tumors don&#39;t have virus.</p></li>
<li><p>Few mutations have been found in oncogenes or tumor suppressors.</p></li>
</ol>

<p><em>Is it the case that the tests for MCV are not sensitive enough, and all tumors have the virus?</em></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Detecting MCV</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure1A.png" width="1000" alt="Figure 1A"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Western Blotting of MCC Cell Lines</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure1B.png" width="500" alt="Figure 1B"></p>

<!---
* Used CM2B4 mouse mAB (detects 2nd exon of LT)
* Generated new antibody (Ab3) to 260 N-terminal residues of LT
* Tested on MCC cell lines (Mkl-1, Mkl-2, MS-1, WaGa, UISO, U-2OS, U2OS+LT)
* Western blot with Ab3 or CM2B4 of cell lysates prepared from MCC cell lines showing a predominant signal for truncated large T antigen between 36 and 50 kDa. U-2OS-LT contains a full-length large T antigen cDNA. Actin blot indicates protein loading.
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>IHC of MCC Sections</h2>
  </hgroup>
  <article>
    <ul>
<li>60 patients, 75 FFPE MCC tumor specimens</li>
</ul>

<p><img src="figure/ihcvirusbinary.png" alt="plot of chunk ihcvirusbinary"> </p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>IHC Staining Intensity Distribution</h2>
  </hgroup>
  <article>
    <p>3: strong; 2: moderate; 1: weak; 0: absent</p>

<p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure2A.png" height="500" alt="Figure 2A"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Virus status by IHC, Original Data</h2>
  </hgroup>
  <article>
    <p><img src="figure/ihcvirusorig.png" alt="plot of chunk ihcvirusorig"> </p>

<!---
* 38 cases where CM2B4 >= 2 were also Ab3 >= 2.
* 50 cases have Ab3 >= 2.
* 9 cases where Ab3  > 0 and CM2B4 = 0.
* Overall higher intensity for Ab3 (only because of samples >= 3)
* Consistent staining within patients
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Virus Positive IHC Stains (Figure 3A)</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure3A.png" width="1000" alt="Figure 3A"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>CM2B4<sup>-</sup> Ab3<sup>+</sup> IHC Stains (Figure 3B)</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure3B.png" width="1000" alt="Figure 3B"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>CM2B4<sup>-</sup> Ab3<sup>-</sup> IHC Stains (Figure 3C)</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure3C.png" height="500" alt="Figure 3C"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>IHC Conclusions</h2>
  </hgroup>
  <article>
    <ul>
<li>Ab3 more sensitive than CM2B4</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>qPCR Detection of MCPyV DNA</h2>
  </hgroup>
  <article>
    <table><thead>
<tr>
<th>Primer</th>
<th>T Antigen Loci</th>
</tr>
</thead><tbody>
<tr>
<td><a href="/mnt/NIH_GROUP/Dermatology/Brownell%20Lab/Kenny/Presentations/BrownellLabMeeting_20121217/doc/references/becker_08_mc_859644.pdf" title="LT2">LT2</a></td>
<td>Large</td>
</tr>
<tr>
<td>LT3</td>
<td>Small</td>
</tr>
<tr>
<td>Set6</td>
<td>Large</td>
</tr>
<tr>
<td>Set7</td>
<td>Large</td>
</tr>
<tr>
<td>Set9</td>
<td>Small</td>
</tr>
</tbody></table>

<p><strong>TPO</strong> was used as copy number control.</p>

<!---
* Two previously developed primer sets (LT2, LT3)
* Three new primer sets (Set6, Set7, Set9)
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>MCV Copy Number in MCC Cell Lines</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure1C.png" width="1000" alt="Figure 1C"></p>

<!---
For validation, ran them on the cell lines (4 MCPyV positive, 2 MCPyV negative, and a positive control).
U-20S w/ LT cDNA amplified using LT primers (LT2, Set6, Set7) but not sT primers (LT3, Set9)
Ct values for Mkl-2 were approx equal to TPO and normalized wrt to this to compare to all other samples.
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>qPCR Copy Number Distribution</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Figure2B.png" height="500" alt="Figure 2B"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>qPCR Copy Number Distribution, per sample</h2>
  </hgroup>
  <article>
    <p><img src="figure/qpcrdistpersample.png" alt="plot of chunk qpcrdistpersample"> </p>

<p>qPCR viral CN as low as 0.03 (from UPI 60) is specific for detection of MCPyV DNA?</p>

<!---
Performed qPCR on the FFPE DNA using LT2, LT3, Set7, TPO in triplicate. If any were less than 1, then Set6 and Set9 were tested.
48 of 60 cases (80%), at least one primer set detected greater than 2 copies per cell, consistent with the clonal integration of MCPyV DNA in every tumor cell
5 cases (UPI 27, 47, 49, 57, and 60) yielded viral copy number values greater than 0.03 but less than 0.1, but still stained with antibodies (very little DNA, but protein!)
2 cases (UPI 15, 42) had CN > 0.1 but were negative with both antibodies (DNA but no protein)
No signal found in control tissues (FFPE tonsillar, reactive lymph nodes, Mkl-2 as positive control)
TPO amplified fine in all samples with signal similar to Mkl-2.
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Correlation of qPCR Copy Number and IHC intensity</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Rodig_Table1.png" width="1000" alt="Table 1"></p>

<!---
* "Significant" correlation between all combos of antibody and qPCR
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Re-compute Correlation of qPCR Copy Number and IHC intensity</h2>
  </hgroup>
  <article>
    <pre><code>##          Ab3_max    CM2B4      LT2      LT3     Set6     Set7 Set9
## Ab3_max              0.79     0.62     0.45     0.51     0.44 0.43
## CM2B4   4.44e-16              0.69     0.57     0.63     0.54 0.49
## LT2     5.87e-09 3.97e-11              0.81     0.75     0.72 0.67
## LT3     8.76e-05  3.3e-07        0              0.54     0.81 0.82
## Set6     0.00143 7.39e-05 6.76e-08 0.000507              0.46 0.61
## Set7    0.000108 1.29e-06 2.22e-13        0    0.004          0.54
## Set9     0.00907  0.00339 5.13e-06 4.46e-10 5.08e-05 0.000407
</code></pre>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Correlation of log(qPCR Copy Number) and IHC intensity</h2>
  </hgroup>
  <article>
    <pre><code>##          Ab3_max    CM2B4      LT2     LT3     Set6     Set7 Set9
## Ab3_max              0.76     0.65    0.35     0.48     0.37 0.33
## CM2B4   1.91e-14              0.76     0.5     0.63     0.51 0.42
## LT2     8.14e-10  3.2e-14             0.73     0.84     0.68 0.59
## LT3      0.00293 9.35e-06 1.16e-13             0.44     0.84 0.89
## Set6     0.00327 6.06e-05 2.72e-11 0.00558              0.33 0.52
## Set7     0.00144 5.39e-06 2.05e-11       0   0.0428           0.7
## Set9      0.0521   0.0135 8.94e-05 1.2e-13 0.000819 1.03e-06
</code></pre>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>qPCR and IHC Intensity Data (Maximum Ab3)</h2>
  </hgroup>
  <article>
    <p><img src="figure/qpcrihcab3.png" alt="plot of chunk qpcrihcab3"> </p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>qPCR and IHC Intensity Data (CM2B4, 2 ug/ml)</h2>
  </hgroup>
  <article>
    <p><img src="figure/qpcrihccm2b4.png" alt="plot of chunk qpcrihccm2b4"> </p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Discussion Points</h2>
  </hgroup>
  <article>
    <ul>
<li>Ab3 is more sensitive than and as specific as CM2B4</li>
<li>comparable to CM5E1 sT antibody (<a href="/mnt/NIH_GROUP/Dermatology/Brownell%20Lab/Kenny/Presentations/BrownellLabMeeting_20121217/doc/references/JCI46323.v3.pdf" title="CM5E1">detection of 92% (47/51) of MCC</a>)</li>
<li>CN &gt; 0.1 in 55/60 (90%) of MCCs (consistent with clonal integration?)</li>
<li>More primer sets, higher chance of detection</li>
<li>Even at CN of 0.03, still higher than control tissues (and detected by IHC)</li>
<li>Advantage of IHC over qPCR for FFPE samples</li>
<li>Lack of point mutations in 112 genes</li>
<li>Three p53 mutations, two in the MCV negative samples!</li>
<li>Even virus positive tumors may not express T antigen?</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Analysis of MCV Load</h2>
  </hgroup>
  <article>
    <p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Ota_Title.png" width="1000" alt="Ota Title"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Analysis of MCV Load</h2>
  </hgroup>
  <article>
    <p>Investigate the frequency of MCPyV infection in skin tumors (MCC, BCC, AK, BD, etc.)</p>

<p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Ota_Figure1.png" width="1000" alt="Ota Figure 1"></p>

<!---
* Conventional PCR detected MCPyV in 8/9 MCC, 1/46 BCC, 3/52 AK, and no others
* Variability in the fragments detected
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>IHC using CM2B4 (MCC)</h2>
  </hgroup>
  <article>
    <p>All cases found positive by conventional qPCR were also positive by IHC
<img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Ota_Figure2AB.png" width="1000" alt="Ota Figure 2AB"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>IHC using CM2B4 (BCC)</h2>
  </hgroup>
  <article>
    <p>All cases found positive by conventional qPCR were also positive by IHC
<img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Ota_Figure2CDE.png" width="1000" alt="Ota Figure 2CDE"></p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Digital PCR</h2>
  </hgroup>
  <article>
    <p>Changes the problem from comparing Ct values to a counting problem (Poisson statistics)</p>

<p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/digitalPCRexample.png" width="900" alt="Digital PCR Example"></p>

<!---
After PCR amplification, nucleic acids may be quantified by counting the regions that contain PCR end-product, positive reactions.
In conventional PCR, the number of PCR amplification cycles is proportional to the starting copy number (exponential amplification)
Issues with conventional PCR include sub-optimal efficiency, plateau, low concentrations not amplifiable/detectable
Allows larger range of difference detection (not limited to 2-fold change)
--->

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Digital PCR for Viral Copy Number</h2>
  </hgroup>
  <article>
    <p>Used primers to the small T antigen (confirmed with a second primer set?) and RNaseP as a contol</p>

<p><img src="/mnt/NIH_GROUP/Dermatology/Brownell Lab/Kenny/Presentations/BrownellLabMeeting_20121217/reports/presentation/assets/img/Ota_Figure3B.png" height="400" alt="Ota Figure 3B"></p>

<p>The negative IHC results for 1 MCC and 3 AK cases were clearly linked to their low viral loads.</p>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Methylation Status of Skin Tumors</h2>
  </hgroup>
  <article>
    <ul>
<li>Hypermethylation of host DNA has been detected in SV40 polyomavirus-related tumors and cell lines as well as in some MCC</li>
<li>Observed DNA hypermethylation of RASSF1A in 6/9 MCC, 7/46 BCC, 1/52 AK</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

      <slide class="class" id="id" style="background:;">
  <hgroup>
    <h2>Summary</h2>
  </hgroup>
  <article>
    <ul>
<li>100% MCPyV infection frequency (small sample size?)</li>
<li>Loss of LT2 fragment in MCC mets</li>
<li>Differing range of viral load for MCC, AK</li>
<li>Strong correlation between IHC and viral load</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>

  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
<!-- Grab CDN jQuery, fall back to local if offline -->
<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
<script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery-1.7.min.js"><\/script>')</script>
<!-- Load Javascripts for Widgets -->
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
</html>