<!DOCTYPE HTML>
<html lang="en-US">
<head>
	<title>Brownell Lab Meeting</title>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=1274, user-scalable=no">
	<meta name="description" content="Brownell Lab Meeting">
	<meta name="author" content="Kenny Daily">
	<meta name="generator" content="slidify" />
	<!-- LOAD STYLE SHEETS -->
	<link rel="stylesheet" href="libraries/frameworks/shower/themes/ribbon/styles/style.css">
	<link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css">  <link rel="stylesheet" href = "assets/css/ribbons.css">

	<!--
		To apply styles to the certain slides
		use slide ID to get needed elements
		-->
	<style>
		#Cover h2 {
			color:#FFF;
			text-align:center;
			font-size:70px;
			}
		#FitToWidth h2,
		#FitToHeight h2 {
			color:#FFF;
			text-align:center;
			}
	</style> 
</head>
<body class="list">
    <div class="slide " id = "slide-1">
  <div>
    <section>
      <header>
        <h2>
          MCPyV genome
        </h2>
      </header>
      <p><img src="./assets/img/MCV350_layout.png" height="400" alt="MCV350"></p>

<p>Science 2008;319;5866:1096-1100. doi:10.1126/science.1152586, Fig. 3</p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-2">
  <div>
    <section>
      <header>
        <h2>
          MCPyV Infection
        </h2>
      </header>
      <ol>
<li>MCC cells do not typically harbor episomal MCPyV viral DNA</li>
<li>MCPyV is clonally integrated in the tumor chromosomal DNA</li>
<li>Mutations resulting in truncation of large T antigen (LT)</li>
<li>Expression of full length LT promotes viral replication</li>
<li>sT is necessary but not sufficent for tumorigenesis</li>
</ol>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-3">
  <div>
    <section>
      <header>
        <h2>
          Detection of MCPyV
        </h2>
      </header>
      <ul>
<li>Several studies show ~80% of MCCs have virus present, mostly by PCR on FFPE tissues</li>
<li>CN variation is prevalent</li>
<li>IHC LT reported as ~80% positive (CM2B4 antibody)</li>
<li>IHC LT reported as ~90% positive (CM5E1 antibody)</li>
</ul>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-4">
  <div>
    <section>
      <header>
        <h2>
          
        </h2>
      </header>
      <p><img src="./assets/img/Ota_Title.png" width="800" alt="Ota Title"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-5">
  <div>
    <section>
      <header>
        <h2>
          Frequency of MCPyV Infection in Skin Tumors
        </h2>
      </header>
      <p><img src="./assets/img/Ota_Figure1.png" width="600" alt="Ota Figure 1"></p>

<!---
* Conventional PCR detected MCPyV in 8/9 MCC, 1/46 BCC, 3/52 AK, and no others
* Variability in the fragments detected
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-6">
  <div>
    <section>
      <header>
        <h2>
          IHC using CM2B4 (MCC)
        </h2>
      </header>
      <p>All cases found positive by conventional qPCR were also positive by IHC
<img src="./assets/img/Ota_Figure2AB.png" width="800" alt="Ota Figure 2AB"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-7">
  <div>
    <section>
      <header>
        <h2>
          IHC using CM2B4 (BCC)
        </h2>
      </header>
      <p>All cases found positive by conventional qPCR were also positive by IHC
<img src="./assets/img/Ota_Figure2CDE.png" width="800" alt="Ota Figure 2CDE"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-8">
  <div>
    <section>
      <header>
        <h2>
          <a href="https://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/PCR/real-time-pcr/real-time-pcr-applications/digital-pcr/viral-load-quantification.html?icid=fr-dpcr-1" title="Link">Digital PCR</a> (<a href="http://media.invitrogen.com.edgesuite.net/ab/applications-technologies/real-time-pcr/digitalpcr/index.html" title="Video">Video</a>)
        </h2>
      </header>
      <p>Instead of comparing Ct values, just count!</p>

<p><img src="./assets/img/digitalPCRexample.png" width="750" alt="Digital PCR Example"></p>

<!---
After PCR amplification, nucleic acids may be quantified by counting the regions that contain PCR end-product, positive reactions.
In conventional PCR, the number of PCR amplification cycles is proportional to the starting copy number (exponential amplification)
Issues with conventional PCR include sub-optimal efficiency, plateau, low concentrations not amplifiable/detectable
Allows larger range of difference detection (not limited to 2-fold change)
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-9">
  <div>
    <section>
      <header>
        <h2>
          Digital PCR for Viral Copy Number
        </h2>
      </header>
      <p>Used primers to the small T antigen and RNaseP as a contol
<img src="./assets/img/Ota_Figure3B.png" height="400" alt="Ota Figure 3B"></p>

<p>The negative IHC results for 1 MCC and 3 AK cases were clearly linked to their low viral loads.</p>

<!---
confirmed with a second primer set?
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-10">
  <div>
    <section>
      <header>
        <h2>
          Methylation Status of Skin Tumors
        </h2>
      </header>
      <ul>
<li>Hypermethylation of host DNA has been detected in SV40 polyomavirus-related tumors and cell lines as well as in some MCC</li>
<li>Observed DNA hypermethylation of RASSF1A in 6/9 MCC, 7/46 BCC, 1/52 AK</li>
</ul>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-11">
  <div>
    <section>
      <header>
        <h2>
          Summary
        </h2>
      </header>
      <ul>
<li>100% MCPyV infection frequency (small sample size?)</li>
<li>Loss of LT2 fragment in MCC mets</li>
<li>Differing range of viral load for MCC, AK</li>
<li>Strong correlation between IHC and viral load</li>
<li>digital PCR is advantageous</li>
</ul>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-12">
  <div>
    <section>
      <header>
        <h2>
          
        </h2>
      </header>
      <p>##
<img src="./assets/img/Rodig_Title.png" width="800" alt="Rodig Title"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-13">
  <div>
    <section>
      <header>
        <h2>
          Hypothesis
        </h2>
      </header>
      <ol>
<li><p>It is widely thought that MCPyV is a causative oncogenic mechanism, but 20% of MCC tumors don&#39;t have virus.</p></li>
<li><p>Few mutations have been found in oncogenes or tumor suppressors.</p></li>
</ol>

<p><em>Is it the case that the tests for MCPyV are not sensitive enough, and all tumors have the virus?</em></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-14">
  <div>
    <section>
      <header>
        <h2>
          Detecting MCPyV
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure1A.png" width="800" alt="Figure 1A"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-15">
  <div>
    <section>
      <header>
        <h2>
          Detecting MCPyV
        </h2>
      </header>
      <p><img src="./assets/img/Shuda_SuppFig1.png" width="800" alt="Shuda Supp Figure 1"></p>

<p>J Clin Invest. 2011;121(9):3623–3634. doi:10.1172/JCI46323, Supp. Fig. 1</p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-16">
  <div>
    <section>
      <header>
        <h2>
          Western Blotting of MCC Cell Lines
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure1B.png" width="500" alt="Figure 1B"></p>

<!---
* Used CM2B4 mouse mAB (detects 2nd exon of LT)
* Generated new antibody (Ab3) to 260 N-terminal residues of LT
* Tested on MCC cell lines (Mkl-1, Mkl-2, MS-1, WaGa, UISO, U-2OS, U2OS+LT)
* Western blot with Ab3 or CM2B4 of cell lysates prepared from MCC cell lines showing a predominant signal for truncated large T antigen between 36 and 50 kDa. U-2OS-LT contains a full-length large T antigen cDNA. Actin blot indicates protein loading.
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-17">
  <div>
    <section>
      <header>
        <h2>
          IHC on MCC Sections
        </h2>
      </header>
      <p>60 patients, 75 FFPE MCC tumor specimens (not all were stained)</p>

<p><img src="figure/ihcvirusbinary.png" alt="plot of chunk ihcvirusbinary"> </p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-18">
  <div>
    <section>
      <header>
        <h2>
          IHC Staining Intensity Distribution
        </h2>
      </header>
      <p>3: strong; 2: moderate; 1: weak; 0: absent</p>

<p><img src="./assets/img/Rodig_Figure2A.png" height="300" alt="Figure 2A"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-19">
  <div>
    <section>
      <header>
        <h2>
          IHC Staining Intensities, Original Data
        </h2>
      </header>
      <p><img src="figure/ihcvirusorig.png" alt="plot of chunk ihcvirusorig"> </p>

<!---
* 38 cases where CM2B4 >= 2 were also Ab3 >= 2.
* 50 cases have Ab3 >= 2.
* 9 cases where Ab3  > 0 and CM2B4 = 0.
* Overall higher intensity for Ab3 (only because of samples >= 3)
* Consistent staining within patients
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-20">
  <div>
    <section>
      <header>
        <h2>
          Virus Positive IHC Stains (Figure 3A)
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure3A.png" width="750" alt="Figure 3A"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-21">
  <div>
    <section>
      <header>
        <h2>
          CM2B4<sup>-</sup> Ab3<sup>+</sup> IHC Stains (Figure 3B)
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure3B.png" width="800" alt="Figure 3B"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-22">
  <div>
    <section>
      <header>
        <h2>
          CM2B4<sup>-</sup> Ab3<sup>-</sup> IHC Stains (Figure 3C)
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure3C.png" height="400" alt="Figure 3C"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-23">
  <div>
    <section>
      <header>
        <h2>
          Ab3 Control IHC Stains (Figure 4A,B)
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure4AB.png" height="400" alt="Figure 4AB"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-24">
  <div>
    <section>
      <header>
        <h2>
          Ab3 Control IHC Stains (Figure 4C,D,E)
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure4CDE.png" height="400" alt="Figure 4CDE"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-25">
  <div>
    <section>
      <header>
        <h2>
          IHC Conclusions
        </h2>
      </header>
      <ul>
<li>Ab3 more sensitive than and as specific as CM2B4</li>
<li>There are more MCPyV positive MCC tumors than originally thought</li>
<li>Stained one MCC MCPyV positive case with controls</li>
</ul>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-26">
  <div>
    <section>
      <header>
        <h2>
          qPCR Detection of MCPyV DNA
        </h2>
      </header>
      <table><thead>
<tr>
<th>Primer</th>
<th>T Antigen Loci</th>
</tr>
</thead><tbody>
<tr>
<td><a href="/mnt/NIH_GROUP/Dermatology/Brownell%20Lab/Kenny/Presentations/BrownellLabMeeting_20121217/doc/references/becker_08_mc_859644.pdf" title="LT2">LT2</a></td>
<td>Large</td>
</tr>
<tr>
<td>LT3</td>
<td>Small</td>
</tr>
<tr>
<td>Set6</td>
<td>Large</td>
</tr>
<tr>
<td>Set7</td>
<td>Large</td>
</tr>
<tr>
<td>Set9</td>
<td>Small</td>
</tr>
</tbody></table>

<p><strong>TPO</strong> was used as copy number control.</p>

<!---
* Two previously developed primer sets (LT2, LT3)
* Three new primer sets (Set6, Set7, Set9)
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-27">
  <div>
    <section>
      <header>
        <h2>
          MCPyV Copy Number in MCC Cell Lines
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure1C.png" width="800" alt="Figure 1C"></p>

<!---
For validation, ran them on the cell lines (4 MCPyV positive, 2 MCPyV negative, and a positive control).
U-20S w/ LT cDNA amplified using LT primers (LT2, Set6, Set7) but not sT primers (LT3, Set9)
Ct values for Mkl-2 were approx equal to TPO and normalized wrt to this to compare to all other samples.
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-28">
  <div>
    <section>
      <header>
        <h2>
          qPCR Copy Number Distribution
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Figure2B.png" height="400" alt="Figure 2B"></p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-29">
  <div>
    <section>
      <header>
        <h2>
          qPCR Copy Number Distribution, per sample
        </h2>
      </header>
      <p>Is qPCR viral CN as low as 0.03 (from UPI 60) detection of MCPyV DNA?
<img src="figure/qpcrdistpersample.png" alt="plot of chunk qpcrdistpersample"> </p>

<!---
Performed qPCR on the FFPE DNA using LT2, LT3, Set7, TPO in triplicate. If any were less than 1, then Set6 and Set9 were tested.
48 of 60 cases (80%), at least one primer set detected greater than 2 copies per cell, consistent with the clonal integration of MCPyV DNA in every tumor cell
5 cases (UPI 27, 47, 49, 57, and 60) yielded viral copy number values greater than 0.03 but less than 0.1, but still stained with antibodies (very little DNA, but protein!)
2 cases (UPI 15, 42) had CN > 0.1 but were negative with both antibodies (DNA but no protein)
No signal found in control tissues (FFPE tonsillar, reactive lymph nodes, Mkl-2 as positive control)
TPO amplified fine in all samples with signal similar to Mkl-2.
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-30">
  <div>
    <section>
      <header>
        <h2>
          Correlation of qPCR Copy Number and IHC intensity
        </h2>
      </header>
      <p><img src="./assets/img/Rodig_Table1.png" width="800" alt="Table 1"></p>

<!---
* "Significant" correlation between all combos of antibody and qPCR
--->

    </section>
  </div>
</div>
    <div class="slide " id = "slide-31">
  <div>
    <section>
      <header>
        <h2>
          Re-compute Correlation of qPCR Copy Number and IHC intensity
        </h2>
      </header>
      <pre><code>        Ab3_max CM2B4  LT2   LT3  Set6 Set7 Set9
Ab3_max          0.79 0.62  0.45  0.51 0.44 0.43
CM2B4         0       0.69  0.57  0.63 0.54 0.49
LT2           0     0       0.81  0.75 0.72 0.67
LT3           0     0    0        0.54 0.81 0.82
Set6      0.001     0    0 0.001       0.46 0.61
Set7          0     0    0     0 0.004      0.54
Set9      0.009 0.003    0     0     0    0     
</code></pre>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-32">
  <div>
    <section>
      <header>
        <h2>
          qPCR and IHC Intensity Data (Maximum Ab3)
        </h2>
      </header>
      <p><img src="figure/qpcrihcab3.png" alt="plot of chunk qpcrihcab3"> </p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-33">
  <div>
    <section>
      <header>
        <h2>
          qPCR and IHC Intensity Data (CM2B4, 2 ug/ml)
        </h2>
      </header>
      <p><img src="figure/qpcrihccm2b4.png" alt="plot of chunk qpcrihccm2b4"> </p>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-34">
  <div>
    <section>
      <header>
        <h2>
          Discussion Points
        </h2>
      </header>
      <ol>
<li>Immunohistochemistry

<ul>
<li>Ab3 is more sensitive than and as specific as CM2B4, comparable to CM5E1 sT antibody (<a href="/mnt/NIH_GROUP/Dermatology/Brownell%20Lab/Kenny/Presentations/BrownellLabMeeting_20121217/doc/references/JCI46323.v3.pdf" title="CM5E1">detection of 92% (47/51) of MCC</a>)</li>
<li>Advantage of IHC over qPCR for FFPE samples</li>
</ul></li>
</ol>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-35">
  <div>
    <section>
      <header>
        <h2>
          Discussion Points
        </h2>
      </header>
      <ol>
<li>qPCR

<ul>
<li>Higher chance of detection due to more primer sets</li>
<li>Large variation in some samples, doesn&#39;t correspond to IHC</li>
<li>CN &gt; 0.1 in 55/60 (90%) of MCCs (consistent with clonal integration?)</li>
<li>Even at CN of 0.03, still higher than control tissues (and detected by IHC)</li>
</ul></li>
</ol>

    </section>
  </div>
</div>
    <div class="slide " id = "slide-36">
  <div>
    <section>
      <header>
        <h2>
          Discussion Points
        </h2>
      </header>
      <ol>
<li>Mutations

<ul>
<li>Lack of point mutations in 112 genes</li>
<li>Three p53 mutations, two in the MCPyV negative samples!</li>
</ul></li>
<li>Even virus positive tumors (integrated virus) may not express T antigen?</li>
</ol>

    </section>
  </div>
</div>
  <div class="progress">
    <div></div>
  </div>
	<script src="libraries/frameworks/shower/scripts/script.js"></script>
	<!-- LOAD HIGHLIGHTER JS FILES -->
	<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
	<script>hljs.initHighlightingOnLoad();</script>
	<!-- DONE LOADING HIGHLIGHTER JS FILES -->
		<!-- Copyright © 2010–2012 Vadim Makeev — pepelsbey.net -->
	<!-- Photos by John Carey — fiftyfootshadows.net -->
</body>
</html>